alpha/notify-worker/app/config.js

```diff
@@ -0,0 +1,8 @@
+
+module.exports = {
+  mail: {
+    host: 'smtp-server',
+    port: 25,
+    from: 'apvs@kainos.com'
+  }
+}
```

Comment:
Interesting approach, I could see us making config.js files in each of our significant applications, so all config values are in a single easy to find location. The values here could be set by environmental variable or scripted find/replace.
How do you see this working?